api/context.go

```diff
@@ -234,6 +238,47 @@ func (h handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (cw *CorsWrapper) ServeHTTP(
+	w http.ResponseWriter,
+	r *http.Request) {
+	allowedMethods := []string{
+		"POST",
+		"GET",
+		"OPTIONS",
+		"PUT",
+		"PATCH",
+		"DELETE",
+	}
+
+	allowedHeaders := []string{
+		"Accept",
+		"Content-Type",
+		"Content-Length",
+		"Accept-Encoding",
+		"Authorization",
+		"X-CSRF-Token",
+		"X-Auth-Token",
```

Comment:
Sorry 1 more.  Not sure what todo with

```
	HEADER_REQUEST_ID             = "X-Request-ID"
	HEADER_VERSION_ID             = "X-Version-ID"
	HEADER_ETAG_SERVER            = "ETag"
	HEADER_ETAG_CLIENT            = "If-None-Match"
	HEADER_FORWARDED              = "X-Forwarded-For"
	HEADER_REAL_IP                = "X-Real-IP"
	HEADER_FORWARDED_PROTO        = "X-Forwarded-Proto"
	HEADER_TOKEN                  = "token"
	HEADER_BEARER                 = "BEARER"
	HEADER_AUTH                   = "Authorization"
	HEADER_MM_SESSION_TOKEN_INDEX = "X-MM-TokenIndex"
	SESSION_TOKEN_INDEX           = "session_token_index"
```

From client.go

Should those be set in allowedHeaders ?